<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HardwareExpert</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Hex View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTabWidget" name="twHex"/>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="hexInfoLine">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Code View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QTabWidget" name="wtPanel">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Scripts Library</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QTreeWidget" name="twScriptCatalog">
           <column>
            <property name="text">
             <string>Avable Scripts</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Description</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>File</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabFunctions">
        <attribute name="title">
         <string>Functions</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="lwApi">
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::NoDragDrop</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="iconSize">
            <size>
             <width>160</width>
             <height>160</height>
            </size>
           </property>
           <property name="movement">
            <enum>QListView::Static</enum>
           </property>
           <property name="resizeMode">
            <enum>QListView::Adjust</enum>
           </property>
           <property name="layoutMode">
            <enum>QListView::SinglePass</enum>
           </property>
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="gridSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="viewMode">
            <enum>QListView::IconMode</enum>
           </property>
           <property name="uniformItemSizes">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="selectionRectVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pbExecute">
             <property name="text">
              <string>Execute</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="waitLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="runButton">
         <property name="text">
          <string>Compile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Log View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPlainTextEdit" name="consoleView">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pbSaveLog">
         <property name="text">
          <string>Save log...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbClearLog">
         <property name="text">
          <string>Clear log</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_code"/>
    <addaction name="actionNew_hex"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Code"/>
    <addaction name="actionSave_code"/>
    <addaction name="actionSave_code_as"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Hex"/>
    <addaction name="actionSave_hex"/>
    <addaction name="separator"/>
    <addaction name="actionImportFile"/>
    <addaction name="actionExportFile"/>
    <addaction name="separator"/>
    <addaction name="actionClose_all_binary"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionDrivers"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHex">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="actionDiff"/>
    <addaction name="actionSearch_String"/>
    <addaction name="actionFind_all_text_string_16_bit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHex"/>
   <addaction name="menuOptions"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionOpen_Code">
   <property name="text">
    <string>Open code...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_Hex">
   <property name="text">
    <string>Open binary...</string>
   </property>
  </action>
  <action name="actionNew_code">
   <property name="text">
    <string>New code</string>
   </property>
  </action>
  <action name="actionNew_hex">
   <property name="text">
    <string>New binary</string>
   </property>
  </action>
  <action name="actionSave_code">
   <property name="text">
    <string>Save code</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_hex">
   <property name="text">
    <string>Save binary...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave_code_as">
   <property name="text">
    <string>Save code as...</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionImportFile">
   <property name="text">
    <string>Import file...</string>
   </property>
  </action>
  <action name="actionExportFile">
   <property name="text">
    <string>Export file...</string>
   </property>
  </action>
  <action name="actionDrivers">
   <property name="text">
    <string>Drivers...</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search...</string>
   </property>
  </action>
  <action name="actionDiff">
   <property name="text">
    <string>Compare...</string>
   </property>
  </action>
  <action name="actionSearch_String">
   <property name="text">
    <string>Find all text strings (8-bit)</string>
   </property>
  </action>
  <action name="actionClose_all_binary">
   <property name="text">
    <string>Close all binary</string>
   </property>
  </action>
  <action name="actionFind_all_text_string_16_bit">
   <property name="text">
    <string>Find all text string (16-bit)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
